docs/cli.md

```diff
@@ -94,10 +103,80 @@ With `--all` flag, you create config file including all default setting of offic
       executable_methods:
         - method_name: "node_governance_alarm"`
 ```
-
 Also, You can set the home directory of VATZ (default "~/.vatz") with `--home` flag.
 And you can create the new config file with your disired name with `--output` flag
 
+## Start
+Please, ensure completion of the configuration setup in either the `default.yaml` file or your custom configuration YAML file prior to initiating VATZ.
+
+```
+./vatz start --help                 
+start VATZ
+
+Usage:
+   start [flags]
+
+Flags:
+      --config string       VATZ config file. (default "default.yaml")
+  -h, --help                help for start
+      --log string          log file export to.
+      --prometheus string   prometheus port number. (default "18080")
+
+Global Flags:
+      --debug   Enable debug mode on Log.
+      --trace   Enable trace mode on Log.
+
+```
+You can simply start VATZ with command
+```
+~$ ./vatz start 
+```
+This will start VATZ with the default config created from the previous vatz init command. 
+You can set the exact config file with the --config flag, such as:
+```
+~$ ./vatz start --config /root/User/vatz-config.yaml
+```
+An absolute path is mandatory for this operation. 
+To view your logs based on their level, you can add the `--debug` or `--trace` flag accordingly.
```

Comment:
flag -> flags

Reply:
it would be flag not flags, because using `or` between it.